RecoHI/HiEvtPlaneAlgos/src/EvtPlaneProducer.cc

```diff
@@ -209,15 +221,144 @@ class EvtPlaneProducer : public edm::stream::EDProducer<> {
   double maxpt_;
   double minvtx_;
   double maxvtx_;
+  int flatnvtxbins_;
+  double flatminvtx_;
+  double flatdelvtx_;
+  double dzdzerror_;
+  double d0d0error_;
+  double pterror_;
+  double chi2perlayer_;
   double dzerr_;
+  double dzdzerror_pix_;
   double chi2_;
+  int nhitsValid_;
   int FlatOrder_;
   int NumFlatBins_;
   double nCentBins_;
   double caloCentRef_;
   double caloCentRefWidth_;
   int CentBinCompression_;
+  int cutEra_;
   HiEvtPlaneFlatten *flat[NumEPNames];
+  TrackStructure track;
+
+  edm::ESWatcher<HeavyIonRcd> hiWatcher;
+  edm::ESWatcher<HeavyIonRPRcd> hirpWatcher;
```

Comment:
```suggestion
  TrackStructure track_;

  edm::ESWatcher<HeavyIonRcd> hiWatcher_;
  edm::ESWatcher<HeavyIonRPRcd> hirpWatcher_;
```

similar to the above, use trailing underscores for  data members